### ISSUE TITLE
No Google 2FA prompt after installation

### ISSUE THREAD

**Issue by mattbrazier** [Offroadcode/Umbraco-2FA#3]
Hello,

I have installed this plugin on a clean installation of Umbraco 7.6.1 and have completed all required steps as per your documentation. After enabling the plugin on my user account I log out and never receive any prompts to complete Google Authenticator setup.

Umbraco is running in a dedicated application pool on IIS 7.5 with ASP.NET 4.6.1. There is nothing in my application data logs to suggest a problem or GET requests for its files in my transfer log.

Is there any additional setup required to make this plugin work or known compatibility issues?

Regards,
Matt.

**Comment by cssquirrel** [comment]
I've just ran through the process on a fresh Umbraco 7.6.1 install and I wasn't able to recreate your error.

Process:

1. Installed a fresh 7.6.1 installation
2. Installed the Umbraco 2FA package.
3. Added back office permission to an admin user.
4. Refreshed page and visited Umbraco 2FA section.
5. Confirmed the Google Authenticator was set to "Enabled".
6. Logged out.
7. Logged in. Was prompted with the Authenticator set up procedure.

Let me try again with a couple different step changes to see if something unexpected happens.
